src/app/views/orders/order.html

```diff
@@ -47,6 +47,22 @@
               {{vm.subscription.product_pricing.name}}
             </div>
           </div>
+          <div class="row">
+            <div class="col-xs-6">
+              <div class="label-cell" translate>mnoe_admin_panel.dashboard.subscription.price</div>
+            </div>
+            <div class="col-xs-6">
+              <mnoe-product-pricing-price subscription="vm.subscription" selected-currency="vm.subscription.currency">
+            </div>
+          </div>
+          <div class="row" ng-show="vm.subscription.product_pricing.license_based">
```

Comment:
Will this throw an error if there is no product_pricing attached to the subscription? Might want to use `vm.subscription.product_pricing?.license_based` here. If it does need to have it, this applies to everywhere else `vm.subscription.product_pricing` is used as well.

Reply:
Hmm... well I don't get any console errors when the product doesn't have a product_pricing.
It seems that angularjs takes care of this when its in a directive.